src/main/java/org/myrobotlab/programab/MrlSraixHandler.java

```diff
@@ -31,7 +37,23 @@ public MrlSraixHandler(ProgramAB programab) {
 
   @Override
   public String sraix(Chat chatSession, String input, String defaultResponse, String hint, String host, String botid, String apiKey, String limit, Locale locale) {
-    log.debug("MRL Sraix handler! Input {}");
+    log.debug("MRL Sraix handler! Input {}", input);
+
+    try {
+      XmlMapper xmlMapper = new XmlMapper();

```

Comment:
this fails if the input isn't xml ...   this will need to change before this PR can get merged ..

Reply:
its caught as i said before - processing continues after if applicable